torch/utils/tensorboard/summary.py

```diff
@@ -73,59 +73,72 @@ def _draw_single_box(image, xmin, ymin, xmax, ymax, display_str, color='black',
-      metric_dict: A dictionary that saves the name of the metric and its value.
+      hparam_dict: A dictionary that contains names of the hyperparameters
+        and their values.
+      metric_dict: A dictionary that contains names of the metrics
+        and their values.
 
     Returns:
-      The `Summary` proto buffer of Experiment, Session_start_info and Session_end_info
+      The `Summary` protobufs for Experiment, SessionStartInfo and
+        SessionEndInfo
     """
-
-    from tensorboard.plugins.hparams.plugin_data_pb2 import HParamsPluginData, SessionEndInfo, SessionStartInfo
-    from tensorboard.plugins.hparams.api_pb2 import Experiment, HParamInfo, MetricInfo, MetricName, Status
-
-    PLUGIN_NAME = 'hparams'
-    PLUGIN_DATA_VERSION = 0
-
-    EXPERIMENT_TAG = '_hparams_/experiment'
-    SESSION_START_INFO_TAG = '_hparams_/session_start_info'
-    SESSION_END_INFO_TAG = '_hparams_/session_end_info'
+    from tensorboard.plugins.hparams.api_pb2 import (
+        Experiment, HParamInfo, MetricInfo, MetricName, Status
+    )
+    from tensorboard.plugins.hparams.metadata import (
+        PLUGIN_NAME,
+        PLUGIN_DATA_VERSION,
+        EXPERIMENT_TAG,
+        SESSION_START_INFO_TAG,
+        SESSION_END_INFO_TAG
+    )
+    from tensorboard.plugins.hparams.plugin_data_pb2 import (
+        HParamsPluginData, SessionEndInfo, SessionStartInfo
+    )
 
     # TODO: expose other parameters in the future.
-    # hp = HParamInfo(name='lr',display_name='learning rate', type=DataType.DATA_TYPE_FLOAT64, domain_interval=Interval(min_value=10, max_value=100))  # noqa E501
-    # mt = MetricInfo(name=MetricName(tag='accuracy'), display_name='accuracy', description='', dataset_type=DatasetType.DATASET_VALIDATION)  # noqa E501
-    # exp = Experiment(name='123', description='456', time_created_secs=100.0, hparam_infos=[hp], metric_infos=[mt], user='tw')  # noqa E501
+    # hp = HParamInfo(name='lr',display_name='learning rate',
+    # type=DataType.DATA_TYPE_FLOAT64, domain_interval=Interval(min_value=10,
+    # max_value=100))
+    # mt = MetricInfo(name=MetricName(tag='accuracy'), display_name='accuracy',
+    # description='', dataset_type=DatasetType.DATASET_VALIDATION)
+    # exp = Experiment(name='123', description='456', time_created_secs=100.0,
+    # hparam_infos=[hp], metric_infos=[mt], user='tw')
 
     hps = [HParamInfo(name=k) for k in hparam_dict.keys()]
     mts = [MetricInfo(name=MetricName(tag=k)) for k in metric_dict.keys()]
 
     exp = Experiment(hparam_infos=hps, metric_infos=mts)
 
     content = HParamsPluginData(experiment=exp, version=PLUGIN_DATA_VERSION)
-    smd = SummaryMetadata(plugin_data=SummaryMetadata.PluginData(plugin_name=PLUGIN_NAME,
-                                                                 content=content.SerializeToString()))
+    smd = SummaryMetadata(
+        plugin_data=SummaryMetadata.PluginData(plugin_name=PLUGIN_NAME,
```

Comment:
This will have the following when I run flake8 on my machine.
E128 continuation line under-indented for visual indent

Reply:
Give the following a try:

```
smd = SummaryMetadata(
    plugin_data=SummaryMetadata.PluginData(
        plugin_name=PLUGIN_NAME,
        content=content.SerializeToString()
    )
)
```